run.go

```diff
@@ -189,9 +194,44 @@ func writeConfig(cfg *jailConfig) error {
 	return nil
 }
 
+func initSeccomp() error {
+	arch, err := seccomp.GetNativeArch()
+	if err != nil {
+		return err
+	}
+	if arch != seccomp.ArchAMD64 {
+		return fmt.Errorf("native arch %s is not amd64", arch)
+	}
+	act := seccomp.ActErrno.SetReturnCode(int16(unix.EPERM))
+	filter, err := seccomp.NewFilter(act)
+	if err != nil {
+		return err
+	}
+	if err := filter.AddArch(seccomp.ArchX86); err != nil {
+		return err
+	}
+	for _, name := range seccompSyscalls {
```

Comment:
Given that the seccomp policy installation is mandatory, we should at least have a method for allowing users to add syscalls to the allowlist